colossalai/zero/sharded_optim/low_level_optim.py

```diff
@@ -58,7 +58,10 @@ def __init__(
             mp_parallel_mode=ParallelMode.MODEL,
 
     # cpu offload
-            cpu_offload=False):
+            cpu_offload=False,
+
+    # forced dtype
+            forced_dtype=None):
```

Comment:
Why do we need this？

Reply:
if we need to integrate to lightning, we need a way to force the optimizer in a current  dtype.